SPECS/cloud-init/cloud-init.spec

```diff
@@ -1,7 +1,7 @@
 %define cl_services cloud-config.service cloud-config.target cloud-final.service cloud-init.service cloud-init.target cloud-init-local.service
 Summary:        Cloud instance init scripts
 Name:           cloud-init
-Version:        22.1
+Version:        22.2
```

Comment:
You'll need to update [cloud-init.signatures.json](https://github.com/microsoft/CBL-Mariner/blob/d525cbe5669e357dd8fc2f2584db8fafbdf5791c/SPECS/cloud-init/cloud-init.signatures.json#L4) with the sha256sum of the new source tar so that our SRPM packing tools can verify that it's what we expect.

> "cloud-init-22.2.tar.gz": "240b170d0241f10fd1eb0d09c8584ee4ec68ef71418af41bdef8b32ea37856dd"

Reply:
Done, thanks!